community/github/pull-requests.md

```diff
@@ -0,0 +1,211 @@
+# 2. pyOpenSci pull request process
+
+When possible, pull request and issue submissions and reviews should follow standard open source workflows. Below are guidelines for pull requests.
+
+## Pull Requests
+
+New pull requests should:
+
+* be created from a fork rather than the pyOpenSci owned parent repository.
+* never be submitted from the `main` branch of your fork.
+
+Authors of new pull requests should whenever possible, do their best to create clean pull requests. 
+
+A clean pull request:
+
+* is focused, and easy to review,
```

Comment:
I'd split this into two bullet points, and add a bit of clarification on what "focused" and "easy to review" each mean.